code/WorkInProgress/Cael_Aislinn/Rust/core_field.dm

```diff
@@ -427,21 +490,27 @@ Deuterium-tritium fusion: 4.5 x 10^7 K
 			AddParticles(reactant, reactants_reacting_pool[reactant])
 //			to_chat(world, "retained: [reactant], [reactants_reacting_pool[reactant]]")
 
-/obj/effect/rust_em_field/Destroy()
+/obj/machinery/power/rust_em_field/Destroy()
 	//radiate everything in one giant burst
 	for(var/obj/effect/rust_particle_catcher/catcher in particle_catchers)
 		qdel(catcher)
 
-	owned_core.owned_field = null
-	owned_core = null
-
 	RadiateAll()
 
 	processing_objects.Remove(src)
```

Comment:
You're no longer an effect and using the power machinery process remove this line.